### PR TITLE
Signin/out BDD refactor

### PR DESCRIPTION
**Pull request by kellyrob99**
- apply bdd style to test and add explicit checks for confirmation messages

### CODE REVIEW

File: testsuite/functional-testsuite/src/test/ft-resources/testsuite-oss/framework/signin_signout.t.js [hunk 1]
```diff
@@ -13,20 +13,38 @@
...
   t.diag('Verify basic sign-in/sign-out');
 
   t.ok(Ext, 'Ext is here');
   t.ok(NX, 'NX is here');
 
-  t.chain(
-      { waitFor: 'stateReceived' },
-      { waitFor: 'userToBeSignedOut' },
+  t.describe('Can login and out of the application', function(t) {
+    var message = t.spyOn(NX.application.getController('Message'), 'addMessage').and.callThrough()
```
Comment (kellyrob99) [inline testsuite/functional-testsuite/src/test/ft-resources/testsuite-oss/framework/signin_signout.t.js]: @jdillon easy enough to verify messages indirectly like this

**Comment by mcculls** [comment]
+1 assuming CI is happy

**Comment by kellyrob99** [comment]
CI with unrelated failure: http://bamboo.s/browse/NX3-OSSF555-1

**Comment by bradbeck** [comment]
+1

**Pull request merged**